osu.Game/Overlays/Chat/ChatTabItemCloseButton.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics;
+using osu.Framework.Input;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using OpenTK;
+using OpenTK.Graphics;
+
+namespace osu.Game.Overlays.Chat
+{
+    public class ChatTabItemCloseButton : OsuClickableContainer
```

Comment:
Why is this in its own class when the consumers are private? I'd make everything public, and probably put them in a `Tabs` namespace.

Reply:
Also why is there both a `ChatTabItemCloseButton` and a `CloseButton`?